docs/spec.md

```diff
@@ -952,23 +953,24 @@ Performs element-wise atan2 operation on `lhs` and `rhs` tensor and produces a
 
 * For floats: `atan2` from IEEE-754.
 * For complex numbers: complex atan2.
+* For quantized types: `dequantize_op_quantize(atan2, lhs, rhs, type(result))`.
 
 #### Inputs
 
-| Label | Name  | Type                                     | Constraints |
-|-------|-------|------------------------------------------|-------------|
-| (I1)  | `lhs` | tensor of floating-point or complex type | (C1)        |
-| (I2)  | `rhs` | tensor of floating-point or complex type | (C1)        |
+| Label | Name  | Type                                                                    | Constraints |
+|-------|-------|-------------------------------------------------------------------------|-------------|
+| (I1)  | `lhs` | tensor of floating-point or complex type or per-tensor quantized tensor | (C1)        |
```

Comment:
The atan2 quantization support isn't ready at the TFLite ecosystem. As far as I know, there will low possibility to use quantization on the atan2 quantization.

In this change, what is our strategy in StableHLO to define the quantization support of each operator?

Reply:
Hi @abattery 

>> what is our strategy in StableHLO to define the quantization support of each operator?

IMO the strategy is best described in the description and hence quoting that

>> Overall, we propose treating quantized elements as floating-point
elements, and therefore ops on quantized tensors as ops on floating-point
tensors, along the lines of dequantize -> float computation -> quantize. This
principle works for most elementwise ops, although there are some exceptions ...

I agree that there might be some ops which might not be use for quantization. In this particular case, I  acknowledge that the PR prefers ease of specification + generality over narrowing down the list of candidates.

>> there will low possibility to use quantization on the `atan2` quantization.

We found that, like for most of the element wise ops, the above applies to `tanh` as well and hence thus we went ahead to proposing its specification. 

Nevertheless, I would like to understand if there is a fundamental problem in quantizing `tanh` in the way we proposed. Please let me know. 

If not, then we might keep it for generality purpose.